tests/features/timeline.feature

```diff
@@ -14,9 +14,8 @@ Feature: Timeline paragraph.
     # The first widget has both fields required.
     When I press "Save"
     Then I should see the following error messages:
-      | error messages           |
-      | Label field is required. |
-      | Title field is required. |
+      | error messages                 |
+      | This value should not be null. |
```

Comment:
This is very wrong: the test was showing the expected behaviour and should have not been changed. It is pointing out the the changes done above are wrong.

Reply:
Indeed, the first item should have the label and title fields required. Restored.